cantools/database/can/c_source.py

```diff
@@ -536,9 +536,13 @@
 
 class Signal(object):
 
-    def __init__(self, signal):
+    def __init__(self, signal, disable_snake_case_conversion):
```

Comment:
it is probably slightly better to use a "positive" name for the parameter, e.g. `use_raw_names`?